hw2/05-form.js

```diff
@@ -1 +1,50 @@
 // Add your code here
+const form = document.getElementById("form");
+
+document.getElementById("resetBtn").addEventListener("click", clearForm);
+let userName = document.getElementById("userName");
+let userEmail = document.getElementById("userEmail");
+let regStatus = document.getElementById("regStatus");
+let progLangChecked = document.getElementById("progLang");
+let opSysChecked = document.getElementById("opSys");
+let fsWebDevChecked = document.getElementById("fsWebDev");
+let otherInfo = document.getElementById("otherInfo");
+
+function logSubmit() {
+  console.log(userName.value);
+  console.log(userEmail.value);
+  console.log(regStatus.value);
+  console.log(progLangChecked.checked);
+  console.log(opSysChecked.checked);
+  console.log(fsWebDevChecked.checked);
+  console.log(otherInfo.value);
+}
+function clearForm() {
+  if (userName.value != null) {
```

Comment:
Airbnb style guides suggest we use !== over !=;  I still struggle with this too :)